starrocks/create.sql

```diff
@@ -104,7 +104,7 @@ CREATE TABLE hits (
     RefererHash BIGINT NOT NULL,
     URLHash BIGINT NOT NULL,
     CLID INT NOT NULL
-)  
-DUPLICATE KEY (CounterID, EventDate, UserID, EventTime, WatchID) 
-DISTRIBUTED BY HASH(UserID) BUCKETS 192
+)
+DUPLICATE KEY (CounterID, EventDate, UserID, EventTime, WatchID)
```

Comment:
There is a section about tuning in the instructions:
https://github.com/ClickHouse/ClickBench#installation-and-fine-tuning

Maybe we should include stock Starrocks and tuned Starrocks. (something similar is done for ClickHouse, see clickhouse/results/*)